_projects/convert_bear_to_aspects.md

```diff
@@ -17,7 +17,8 @@ name: "Convert Bears to Aspects"
 requirements:
   - "The participant should have one patch to any bear accepted."
   - "The participant should have understanding of [cEP-0005](https://github.com/coala/cEPs/blob/master/cEP-0005.md)."
-  - "The proposal should contain bears which are planned to be addressed and a rough draft on which aspects will be needed and how they will be implemented."
+  - "The proposal should contain bears which are planned to be addressed and a rough draft on which aspects will be
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.